umm-spec-lib/test/cmr/umm_spec/test/migration/version/collection.clj

```diff
@@ -1680,3 +1683,52 @@
              :Coordinate1 {:MinimumValue 1, :MaximumValue 10}}
             {:TilingIdentificationSystemName "WRS-1",
              :Coordinate1 {:MinimumValue 1, :MaximumValue 10}}]))))
+
+(def related-urls-UMM-1-10-example
+  (remove-empty-maps
+    (js/parse-umm-c
+      (assoc exp-conv/example-collection-record-edn
+             :RelatedUrls [{:Description "Related url description"
+                            :URL "http://www.foo.com?a=1&ver=5"
+                            :URLContentType "DistributionURL"
+                            :Type "GET DATA"
+                            :Subtype "EARTHDATA SEARCH"
+                            :GetData {:Format "ascii"
+                                      :MimeType "application/json"
+                                      :Checksum "checksum"
+                                      :Size 10.0
+                                      :Unit "MB"
+                                      :Fees "fees"}}
+                           {:Description "Related url 3 description "
+                            :URL "http://www.foo.com"
+                            :URLContentType "DistributionURL"
+                            :Type "GET SERVICE"
+                            :GetService {:MimeType "application/json"
+                                         :DataID "dataid"
+                                         :DataType "datatype"
+                                         :Protocol "HTTP"
+                                         :FullName "fullname"
+                                         :Format "ascii"
+                                         :URI ["http://www.foo.com", "http://www.bar.com"]}}
+                           {:Description "Related url 2 description"
+                            :URL "http://www.foo.com"
+                            :URLContentType "VisualizationURL"
+                            :Type "GET RELATED VISUALIZATION"
+                            :Subtype "GIBS"}]))))
+
+(deftest migrate-1-10-to-1-11
+  (let [result (vm/migrate-umm {} :collection "1.10" "1.11" related-urls-UMM-1-10-example)]
+    (is (= (remove-empty-maps exp-conv/example-collection-record)
```

Comment:
I'm a little worried about these remove-empty-maps calls. Maybe we should pair and work through this to see why these are needed, and if there is a better solution.

Reply:
Thanks for paring with me and finding a better solution.